skribenten-web/bff/.nais/bff-prod.yaml

```diff
@@ -0,0 +1,25 @@
+app-name: "skribenten-web"
+
+ingresses:
+  - "https://pensjon-skribenten.intern.nav.no"
+
+env:
+  - name: NODE_ENV
+    value: "production"
+  - name: EXPRESS_HOST
+    value: "::"
+  - name: EXPRESS_PORT
+    value: "8080"
+  - name: SKRIBENTEN_API_URL
+    value: "http://skribenten-backend"
```

Comment:
Hmm.. Siden vi bruker service host mot backend så trenger vel faktisk ikke backend en ingress :thinking:

Reply:
Hm, hva mener du den heller skulle være?

Reply:
Det var ikke en kommentar på at du skal endre noe, men om vi skal fjerne ingress for skribenten-backend. Ikke nødvendig å eksponere den om den egentlig ikke skal brukes.

Reply:
ah, da er jeg med